config.js

```diff
@@ -1,6 +1,6 @@
 module.exports = {
-  "name": "Friendly Key Backup Service",
-  "serviceurl": "http://keyrecoveryservice.yourdomain.com/",
+  "name": "BitGo Key Recovery Service",
```

Comment:
Reminder that this is an example project that our partners and the greater ecosystem may have use for. I think our defaults should still look like dummy names, and we should configure our instance of KRSv2 with our own branding

Reply:
ah good point!